hugo/assets/scripts/mer.ts

```diff
@@ -0,0 +1,2 @@
+export * from "@typefox/monaco-editor-react";
```

Comment:
`mer`? Ah! `monaco-editor-react`?! Please do not use abbreviations, it requires readers to look it up or guess.